svc-bip-api/src/main/resources/application-test.yml

```diff
@@ -35,15 +35,14 @@ lhAPIProvider:
   validateToken: "NO"
 
 bip:
-  claimBaseUrl: localhost
+  claimBaseUrl: ${BIP_CLAIM_URL}
```

Comment:
I think this still needs to be `localhost:20300` because locally, when `source scripts/setenv.sh` sets the `BIP_CLAIM_URL` to `mock-bip-claims-api:2030` which is great if your running the integration test within `vro_intranet`. But the integration test isnt, so its unable to resolve that hostname.

Reply:
Also, if this isn't the case, I wonder whats different between our setups? 🤔

Reply:
I think we should update and set  `BIP_CLAIM_URL` value to `localhost:20300` in our local secret folder